registry/flutter_cocoon.test

```diff
@@ -0,0 +1,7 @@
+contact=flutter-infra@google.com
+fetch=git clone https://github.com/flutter/cocoon.git tests
+fetch=git -C tests checkout b5f4a23ed535ce4fed77e3c1124643e0838f6fdc
```

Comment:
Do we have any plans to bump the cocoon version as time goes?

Reply:
flutter/tests unfortunately doesn't have an auto-roller set up, but @Hixie or @godofredoc might have something in the works. For now, we manually bump whenever there's a breakage (it's relatively infrequent, every few months last time we were doing it)